components/datetime/src/skeleton.rs

```diff
@@ -375,11 +375,12 @@ pub fn create_best_pattern_for_fields<'a>(
     let FieldsByType { date, time, other } = group_fields_by_type(&fields);
 
     if !other.is_empty() {
-        // TODO(#418) - TimeZones
+        // These require "append items" support, see #586.
+        // TODO(#583) - TimeZones
         // TODO(#486) - Eras,
         // ... etc.
-        unimplemented!(
-            "There are no \"other\" fields supported, these need to be appended to the pattern."
+        eprintln!(
+            "There are no \"other\" fields supported, these need to be appended to the pattern. {:?}", other
```

Comment:
`question`: this doesn't seem to be related to the bustage and it is a functional change from a panic to a print to stderr. Why did you put it in this PR? I'm not sure if this is the right change tbh.

Reply:
I assumed this branch was going to be hit by my changes, but it's still not. See my latest commit for how I've addressed this question. I think the most technically correct solution would be to return a `BestSkeleton::MissingOrExtraFields` when a time zone is used, but I'm not sure it's worth the code diff to accomplish that. The components bag API is throwing away this information currently:

```
            match skeleton::create_best_pattern_for_fields(
                &self.date_time.skeletons,
                &self.date_time.length_patterns,
                &requested_fields,
            ) {
                skeleton::BestSkeleton::AllFieldsMatch(pattern)
                | skeleton::BestSkeleton::MissingOrExtraFields(pattern) => Some(pattern),
                skeleton::BestSkeleton::NoMatch => None,
            },
```

Reply:
yeah, I understand and am comfortable with WIP situation. I was uncertain about switching a panic to a println only. I don't think we should have any println or eprintln in the lib at this point.

Reply:
Is there a follow-up issue for this?

I prefer keeping this PR small to fix the broken build.